backend/core/package.json

```diff
@@ -19,6 +19,10 @@
     "db:migration:run": "yarn typeorm migration:run",
     "db:migration:generate": "yarn typeorm migration:generate",
     "db:reseed": "yarn db:drop && yarn db:create && yarn db:add-uuid-extension && yarn db:migration:run && yarn db:seed",
+    "db:detroit:reseed":"yarn db:reseed && yarn db:detroit:delist && yarn db:detroit:deunit && yarn db:detroit:seed",
+    "db:detroit:delist":"psql -d bloom -c 'DELETE FROM listings;'",
+    "db:detroit:deunit":"psql -d bloom -c 'DELETE FROM units;'",
```

Comment:
I think we can drop "detroit" from the name of these commands, since they're not specific to Detroit data. Also could we change the names to something like "db:delete-listings" and "db:delete-units"?

Reply:
I have updated the delete scripts to be db:delete-[table name]